Doc/library/os.rst

```diff
@@ -3781,6 +3781,425 @@ features:
    .. versionadded:: 3.10
 
 
+Timer File Descriptors
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 3.13
+
+These functions provide support for Linux's *timer file descriptor* API.
+Naturally, they are all only available on Linux.
+
+The following example shows how to use a timer file descriptor
+to execute a function twice a second:
+
+.. code-block:: python
+
+   # Practical scripts should use really use a non-blocking timer,
+   # we use a blocking timer here for simplicity.
+   import os, time
+
+   # Create the timer file descriptor
+   fd = os.timerfd_create(time.CLOCK_REALTIME)
+
+   # Start the timer in 1 second, with an interval of half a second
+   os.timerfd_settime(fd, initial=1, interval=0.5)
+
+   try:
+       # Process timer events four times.
+       for _ in range(4):
+          # read() will block until the timer expires
+          _ = os.read(fd, 8)
+          print("Timer expired")
```

Comment:
It looks to me like 3-space indentation instead of PEP 8 recommended 4-space indentation.

Reply:
@serhiy-storchaka 
I'll fix it.

The indent of `.. code-block:: python` is 3-space.
So I'm confused.

Reply:
@serhiy-storchaka 
Fixed the indent at f06e3525b3d1ae2f59d0adfba00279d398e3fbd3.